### DISCUSSION TITLE
Android build?

### DISCUSSION THREAD

**Discussion by ripsivis** [Alex313031/thorium#113] [General]
Chromium forks for Android are rather incomplete, bloated, lacking from basic features, closed-source or somehow broken. Having a thorium build for android with features like extension support would be exciting.

**Comment by ripsivis** [comment]
Thoughs on some chromium based browsers

Brave:
+ Adblocker is powerful
+ Has good privacy options

- Bloated, mostly with crypto & web3 stuff
- Doesn't support material you theming
- Doesn't support extensions
- Browser itself is generally broken 

Kiwi:
+ Has some cool stuff like devtools & extensions

- Closed-source
- Bloated
- Doesn't support material you theming

Bromite:
- Having problems with some pages
- Adblocker exists but it feels incomplete

Ungoogled Chromium
- Too much debloated
- [Extension support](https://github.com/wchen342/chromium-android-extension) doesn't exists anymore

**Reply by Universalizer** [reply]
Ungoogled Chromium latest version, is it available for Android ?

**Reply by Universalizer** [reply]
https://github.com/wchen342/chromium-android-extension

https://github.com/wchen342/ungoogled-chromium-android/releases
last released from Jun 8, 2020

**Reply by onurmercury** [reply]
https://github.com/macchrome/droidchrome/releases

**Reply by Universalizer** [reply]
> https://github.com/macchrome/droidchrome/releases

`Previously,` Fork of Bromite, assuming `Now` of Cromite.
https://github.com/Universalizer/DroidChrome

**Reply by Universalizer** [reply]
> Ungoogled Chromium
Too much debloated

It is pro (advantage) or con ?

> [Extension support](https://github.com/wchen342/chromium-android-extension) doesn't exists anymore

Can be taken, as base, for extension support.<hr>

But, Developers and End-users, please check these concerns, https://github.com/uazo/cromite/issues/256

**Comment by Insidevk18** [comment]
Some notes to be taken into account:
* Brave android now allows to add custom lists.

* Bromite was abandoned, its fork Cromite, has considerably improved AdBlock, and also allows custom lists.

* Vivaldi browser android has a good ad blocker, and also allows custom lists. The bad thing is that the browser comes with too many features in my opinion unnecessary.

* Yandex Browser allows you to install certain browser extensions, apparently you can install ghostery adblocker, but Yandex comes bloated with too much bloatware.

**Comment by gz83** [comment]
Already have the Android version
